core/src/main/java/org/bitcoinj/evolution/QuorumRotationState.java

```diff
@@ -336,8 +333,7 @@ public void applyDiff(Peer peer, AbstractBlockChain headersChain, AbstractBlockC
             SimplifiedQuorumList newQuorumListAtHMinusC = baseQuorumList.applyDiff(quorumRotationInfo.getMnListDiffAtHMinusC(), isLoadingBootStrap, chain, true, false);
 
             baseQuorumList = quorumsCache.get(quorumRotationInfo.getMnListDiffAtH().prevBlockHash);
-            // only verify quorums if extraShare == true, otherwise we don't have all the necessary mn lists
-            SimplifiedQuorumList newQuorumListAtH = baseQuorumList.applyDiff(quorumRotationInfo.getMnListDiffAtH(), isLoadingBootStrap, chain, true, quorumRotationInfo.hasExtraShare());
+            SimplifiedQuorumList newQuorumListAtH = baseQuorumList.applyDiff(quorumRotationInfo.getMnListDiffAtH(), isLoadingBootStrap, chain, true, false);
```

Comment:
This change will skip verifying of old quorums, which adds several seconds of processing on mobile devices.